### PR TITLE
Remove include_chain_clauses_in_custom_matcher_descriptions

### PR DESCRIPTION
**Pull request by pirj**

### CODE REVIEW

File: lib/rspec/core/project_initializer/spec/spec_helper.rb [hunk 1]
```diff
@@ -18,14 +18,6 @@
   # assertion/expectation library such as wrong or minitest
   # assertions if you prefer.
   config.expect_with :rspec do |expectations|
```
Comment (pirj) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb:20]: This looks empty now, maybe put this here?
```ruby
  expectations.max_formatted_output_length = 200
```
It proved to be quite useful.

Comment (JonRowe) [reply lib/rspec/core/project_initializer/spec/spec_helper.rb:20]: 👍  with a comment saying the 200 is the default

**Review by JonRowe — APPROVED** [review]

File: lib/rspec/core/project_initializer/spec/spec_helper.rb [hunk 2]
```diff
@@ -18,14 +18,9 @@
...
-    # This option will default to `true` in RSpec 4. It makes the `description`
-    # and `failure_message` of custom matchers include text for helper methods
-    # defined using `chain`, e.g.:
-    #     be_bigger_than(2).and_smaller_than(4).description
-    #     # => "be bigger than 2 and smaller than 4"
-    # ...rather than:
-    #     # => "be bigger than 2"
-    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+    # 200 is the default value. You may want to increase it to make the
+    # output more verbose.
```
Comment (JonRowe) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb:22]:
```suggestion
    # The default output length is 200, you can increase it to make the
    # output more verbose, or decrease it to make it more concise.
```

**Review by JonRowe — APPROVED** [review]

**Comment by JonRowe** [comment]
Merge on 🟢

**Pull request merged**